### PR TITLE
initialization

### PR DESCRIPTION
**Pull request by greegouze**
## Description

Please include a summary of the change or which issue is fixed.
Pull Request initialization

## Type of change

- Add variables
- Add fonts
- Import Tailwind
- Add dark mode

## Area of change

- [ ] Other: .....

## General checklist:

- [X] My code follows the style guidelines of this project
- [X] I ran `bun run format`/`bun run lint` before commit
- [X] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [X] I have made corresponding changes to the documentation where needed
- [X] My changes generate no new warnings
- [X] I have checked my code and corrected any misspellings
- [X] I have updated `develop` and merged to my branch before submitting pull request
- [X] My pull request generate no conflicts with `develop` branch
- [X] I requested code review from other team members

## Frontend checklist:

- [X] I followed guidelines for `HTML`/`JSX`/`Typescript`
- [X] My `Javascript` generate no new console errors
- [X] I tested my code cross browsers
- [X] My slice is pixel perfect for both desktop and mobile according to design
- [X] I conducted basic QA to assure all features are working
- [ ] I tested responsive for mobile and tablet resolutions

## Backend checklist:

- [ ] I tested admin by manually adding content from zero
- [ ] I created easy to use admin experience which is self-explanatory
- [ ] I added description to admin fields in hard-to-understand areas
- [ ] I followed guidelines for naming `Typescript` variables
- [ ] I conducted basic QA to assure all features are working

### CODE REVIEW

**Review by Mikazian — APPROVED** [review]

File: packages/frontend/web/tailwind.config.js [hunk 1]
```diff
@@ -0,0 +1,34 @@
+import { title } from 'process';
```
Comment (YDrogen) [inline packages/frontend/web/tailwind.config.js:1]: Pourquoi ?

File: packages/frontend/web/src/pages/Home.tsx [hunk 2]
```diff
@@ -3,6 +3,10 @@ import { useEffect, useState } from 'react';
 import { useDisclosure } from '@app/frontend-shared';
 import type { SomeInterface, User } from '@app/types';
 
+import ThemeSwitcher from '@/components/ThemeSwitcher';
+
+import '../globals.css';
```
Comment (YDrogen) [inline packages/frontend/web/src/pages/Home.tsx:8]: Pourquoi ?

**Review by YDrogen — CHANGES_REQUESTED** [review]
❌ -> `main`
✅ -> `develop`

**Review by Mikazian — DISMISSED** [review]

**Review by Mikazian — DISMISSED** [review]

**Review by ElBarto444 — APPROVED** [review]
c'est good pour moi

**Review by Mikazian — APPROVED** [review]

**Comment by Mikazian** [comment]
PR corrigée et bien push sur la branche develop !

**Review by ElBarto444 — APPROVED** [review]

**Review by YDrogen — APPROVED** [review]

**Pull request merged**